plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -831,8 +832,12 @@ protected String mount(String path, String parent, Integer nfsVersion) {
         String result = null;
         Script command = new Script(true, "mount", _timeout, s_logger);
         command.add("-t", "nfs");
-        if (nfsVersion != null){
-            command.add("-o", "vers=" + nfsVersion);
+        if (nfsMajorVersion != null){
+            String nfsVersion = "vers=" + nfsMajorVersion;
+            if (nfsMinorVersion != null && nfsMajorVersion >= 4) {
```

Comment:
Is minor version only supported for version 4 and above? If so, why not simply parse and pass the major version when it is less than `3`?

Reply:
This code does exactly that, I don't understand what you would want to change.

Reply:
@Spaceman1984 why not simply pass the version (a single variable) than two?

Reply:
I'll change it to 1 field